h2o-py/h2o/frame.py

```diff
@@ -3435,10 +3436,12 @@ def distance(self, y, measure=None):
     def drop_duplicates(self, columns, keep="first"):
         """
         Drops duplicated rows across specified columns.
+        
         :param columns: Columns to compare during the duplicate detection process.
         :param keep: Which rows to keep. Two possible values: ["first", "last"]. The "first" value (default) keeps
          the first row and deletes the rest. The "last" value keeps the last row.
-        :return: A new H2OFrame with rows deduplicated
+         
+        :returns: A new H2OFrame with rows deduplicated
```

Comment:
are you encouraging us to use `returns` instead of `return`? Afaik both are valid and I tend to use the latter, but for the sake of consistency, I'm ready to use the conjugated version and pay attention to this when reviewing other's PR as well.

Reply:
Okay so when I first started working with the python docs I only ever saw `:returns:` so I thought that it was standard and it was like halfway through doing this update that I learned both are valid. 

I do think moving forward for consistency we should stick to `:returns:`, however. I might tack that on to the PR for py doc improvements, too.